### PR TITLE
Safari bug - mh scroll

### PR DESCRIPTION
**Pull request by KarinaMendez2**
for: https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/833

### CODE REVIEW

File: src/app/components/AllOrNone.tsx [hunk 1]
```diff
@@ -24,6 +24,8 @@ export default styled(AllOrNone)`
   &,
   ${ButtonLink} {
     font-size: 1.4rem;
+    margin: 0;
```
Comment (PiotrKozlowski) [inline src/app/components/AllOrNone.tsx]: In the `ChapterFilter` line 115 there is also margin for `AllOrNone` which happens to have bigger priority so this `margin: 0` is making a difference.
I think you can remove this line, it looks good without this.

File: src/app/components/AllOrNone.tsx [hunk 2]
```diff
@@ -24,6 +24,8 @@ export default styled(AllOrNone)`
...
+    overflow: hidden;
```
Comment (PiotrKozlowski) [inline src/app/components/AllOrNone.tsx]: Can this be `overflow: visible;`?
I've spotted this issue also on my pr: https://github.com/openstax/rex-web/pull/525 in the Edge browser and there only `overflow: visible` works

**Review by PiotrKozlowski — CHANGES_REQUESTED** [review]

File: src/app/components/AllOrNone.tsx (hunk 2 above)
Comment (KarinaMendez2) [reply src/app/components/AllOrNone.tsx]: image

this overflow is what brakes the buttons in safari, `overflow: hidden` fixes it

Comment (KarinaMendez2) [reply src/app/components/AllOrNone.tsx]: I guess we can make it visible too

**Review by PiotrKozlowski — APPROVED** [review]

**Pull request merged**